make/modules/java.base/Lib.gmk

```diff
@@ -228,6 +228,7 @@ ifeq ($(ENABLE_FALLBACK_LINKER), true)
       LDFLAGS := $(LDFLAGS_JDKLIB) \
                  $(call SET_SHARED_LIBRARY_ORIGIN), \
       LIBS := $(LIBFFI_LIBS), \
+	  LIBS_windows := $(LIBFFI_LIBS) ws2_32.lib, \
```

Comment:
Indentation looks off. Make sure you aren't mixing in tabs here.

Reply:
Good catch, I indeed had a tab in there (I have github set to show tabs as 4 spaces, so didn't notice this).